app/assets/stylesheets/helpers/_header.scss

```
@@ -483,21 +483,21 @@ $light-blue: #259EDA;
     @extend %site-width-container;
     position: relative;
 
-    @include media($max-width: 320px) {
+    @include govuk-media-query($until: mobile) {
       .dismiss {
         position: relative;
       }
     }
 
-    @include media(tablet) {
+    @include govuk-media-query($from: tablet) {
       .dismiss {
         position: absolute;
         right: 0;
         top: 0;
       }
     }
 
-    @include media($min-width: 320px) {
+    @include govuk-media-query($until: tablet) {
```

This possibly should be from mobile?

```suggestion
    @include govuk-media-query($from: mobile) {
```